src/usdb_syncer/gui/forms/CommentDialog.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is generated and should not be checked in.

Reply:
How is this generated? I might not have the tooling for this installed

Reply:
It already got generated. The issue is that it's checked in. You have to `git rm` it.

Reply:
Removed this file